Calibration/HcalAlCaRecoProducers/src/AlCaIsoTracksProducer.cc

```diff
@@ -458,15 +468,21 @@ AlCaIsoTracksProducer::select(edm::Handle<edm::TriggerResults>& triggerResults,
 						  trkCaloDirections,
 						  a_charIsoR_,
 						  nNearTRKs, false);
+      HcalDetId detId = (HcalDetId)(trkDetItr->detIdHCAL);
+      int       ieta = detId.ietaAbs();
+      double eIsolation = (useEtaVaryingCut_) ? (maxRestrictionP_*exp(slopeRestrictionP_*((double)(ieta)))) : 0.0;
+      if (eIsolation < eIsolate1_) eIsolation = eIsolate1_;
+      if (eIsolation < eIsolate2_) eIsolation = eIsolate2_;
 #ifdef DebugLog
```

Comment:
At the moment we define 3 parameters - the calibration method can have 3 possible selections  and I just translated that. One can make them by using only 2 - eta dependent and eta independent cuts. Is this what is required?